python/mxnet/symbol.py

```diff
@@ -133,13 +143,33 @@ def __copy__(self):
         return self.__deepcopy__(None)
 
     def __deepcopy__(self, _):
+        """Returns a deep copy of the input object.
+
+        This functions takes an object as an input and returns a deep copy of that object
+        (i.e. not just a pointer to) including the current state of its parameters
+        (e.g. weight, biases etc. if any).
+
+        Any changes made to a deep copy of object do not reflect in the original object.
```

Comment:
Reword as: "Any changes made to the deep copy do not reflect in the original object."